boards/imix/src/test/ipv6_lowpan_test.rs

```diff
@@ -487,6 +487,7 @@ impl<'a, A: time::Alarm<'a>> TxClient for LowpanTest<'a, A> {
     }
 }
 
+#[allow(clippy::cast_ptr_alignment)]
```

Comment:
Is this actually safe here? Misaligned access on some ARM cores will fault. If we're casting structures onto arbitrary byte buffers that could be problematic.

If it is safe, can we scope down this allow? It's covering over 100 lines of code right now, it'd be nice to reduce this to the few line(s) that actually need it.

Reply:
This is not safe, but it is just test code that isn't enabled by default, and seems to work correctly on the one platform that this test is written for. I suppose it should be fixed anyway, as I am not very familiar with the semantics of when misaligned accesses can cause failures, and as this is non-idiomatic.

Reply:
Eh, I think for a test a comment is probably sufficient. I wouldn't invest more time.